### PR TITLE
[@tools-website] Fix cannot close side menu

### PR DESCRIPTION
**Pull request by KristinaSpasevska**

### CODE REVIEW

File: packages/apps/tools/src/components/Common/Layout/partials/Sidebar/MenuButton.tsx [hunk 1]
```diff
@@ -41,8 +45,19 @@ export const MenuButton: FC<IMenuButtonProps> = ({
         {...rest}
         aria-label={title}
       >
-        <Icon size={'sm'} />
+        <span onClick={onClick}>
```
Comment (sanderlooijenga) [inline packages/apps/tools/src/components/Common/Layout/partials/Sidebar/MenuButton.tsx:48]: We can't use `onClick` of button?

File: packages/apps/tools/src/components/Common/Layout/partials/Sidebar/Toolbar.tsx [hunk 2]
```diff
@@ -21,19 +22,27 @@ export const Toolbar: FC = () => {
...
+    setIsMenuOpen,
   } = useLayoutContext();
   const { pathname } = useRouter();
 
-  const handleItemClick = (index: number): void => {
+  const handleItemClick = (
+    event: React.MouseEvent<HTMLButtonElement, MouseEvent>,
+    index: number,
+  ): void => {
+    event.preventDefault();
```
Comment (sanderlooijenga) [inline packages/apps/tools/src/components/Common/Layout/partials/Sidebar/Toolbar.tsx:33]: Out of curiosity; why do we need to do this?

Comment (KristinaSpasevska) [reply packages/apps/tools/src/components/Common/Layout/partials/Sidebar/Toolbar.tsx:33]: to prevent refresh of the page

Comment (sanderlooijenga) [reply packages/apps/tools/src/components/Common/Layout/partials/Sidebar/Toolbar.tsx:33]: Ah, right 💡

**Review by sanderlooijenga — APPROVED** [review]

**Pull request merged**